### PR TITLE
(feat) Add live nonce analysis view

### PR DESCRIPTION
**Pull request by chron0**

### CODE REVIEW

File: liveviz/convertjson.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    diff = []
+
+    for row in blockdata:
+        #print (row)
+        timestamp.append(row[3])
+        height.append(row[1])
+        nonce.append(row[2])
+        diff.append(row[0])
+
+    print ("{")
```
Comment (serhack) [inline liveviz/convertjson.py:28]: Oh, I'm not sure about this. Why aren't you manage all JSON_encoding with json_dumps function?

**Comment by noncesense-research-lab** [comment]
AWESOME

**Comment by serhack** [comment]
@noncesense-research-lab oh! I was reviewing it.

**Comment by serhack** [comment]
I've a question for @chron0 : how are you managing the blockchain.json file with Python script?

**Comment by chron0** [comment]
as the code suggests: ./convertjson.py nonce-input.csv > blockdata.json

**Comment by serhack** [comment]
@chron0 I understood this :- ) I mean, did you copy-paste the output of the python script into blockchain.json ? 

edit: fantastic!

File: liveviz/convertjson.py (hunk 1 above)
Comment (chron0) [reply liveviz/convertjson.py:28]: it's just a simple hack to get it in the format echarts needs. pythons json_dump didn't produce the result I needed or maybe I was using it wrong. If you can optimize this to get rid of this mess in a pythonic way while still keeping the same output format I'd be very happy to learn :)

**Pull request merged**